app/ante/account.go

```diff
@@ -251,6 +222,11 @@ func incrementSeq(ctx sdk.Context, msgEthTx *evmtypes.MsgEthereumTx, accAddress
 	return
 }
 
+func updateInnerTx(ik innertx.InnerTxKeeper, sk types.SupplyKeeper, ctx sdk.Context, fromAcc exported.Account, fees sdk.Coins, err error) {
+	toAcc := sk.GetModuleAddress(types.FeeCollectorName)
```

Comment:
need to check ctx ischecktx。only delivertx can add innertx

Reply:
got it